deployment/charts/values.yaml

```diff
@@ -22,6 +22,8 @@ app:
   max_transmit_size: "${fuel_core_max_buffer_size}"
   sync_max_get_header: "${fuel_core_sync_max_get_header}"
   sync_max_get_txns: "${fuel_core_sync_max_get_txns}"
+  sync_header_batch_size: "${fuel_core_sync_header_batch_size}"
```

Comment:
we can also set defaults for these vars in the `.env` which is used in automated deployments from master: https://github.com/FuelLabs/fuel-core/blob/master/deployment/scripts/.env